rebasehelper/specfile.py

```diff
@@ -30,7 +30,8 @@
 import shlex
 from datetime import date
 from operator import itemgetter
-from six.moves.urllib.parse import urlparse
+# TODO: remove the pylint comment once pylint with https://github.com/PyCQA/pylint/issues/551 is released
+from six.moves.urllib.parse import urlparse  # pylint: disable=import-error
```

Comment:
The `import-error` can be also avoided by importing only `urllib` from `six.moves` and later calling `urllib.parse.urlparse()` instead of just `urlparse()`. Like in [utils.py](/rebase-helper/rebase-helper/blob/81ae814e3056bb7d5050d42394dc0e6cf26b4c38/rebasehelper/utils.py#L44).